database/engine/rrdengineapi.c

```diff
@@ -12,6 +12,9 @@ struct rrdengine_instance multidb_ctx_storage_tier4;
 #endif
 struct rrdengine_instance *multidb_ctx[RRD_STORAGE_TIERS];
 
+uint8_t tier_page_type[] = {PAGE_METRICS, PAGE_TIER, PAGE_TIER, PAGE_TIER, PAGE_TIER};
```

Comment:
I suggest to make this array of size `RRD_STORAGE_TIERS`, to make it obvious (and searchable) that this needs to be updated also when RRD_STORAGE_TIERS is changed (there is already a compiler error to detect this change).